application/src/states/track-loader.js

```diff
@@ -169,16 +213,18 @@ TrackLoaderState.prototype.update = function()
 
     this.track.enforce(this.car);
 
-    if (this.cursors.up.isDown) {
-        this.car.accelerate();
-    } else if (this.cursors.down.isDown) {
-        this.car.brake();
-    }
+    if (! this.car.falling) {
```

Comment:
Instead of having this check here, what do you think of having something internal to the car object that locks out response to inputs while it's falling? If we end up with multiple states that could lock out the car controls, I wouldn't want to have to list them out here.